package.json

```diff
@@ -56,8 +59,8 @@
     "eslint-plugin-no-only-tests": "^3.1.0",
     "eslint-plugin-node": "^11.1.0",
     "ethereum-waffle": "^4.0.10",
-    "ethers": "^6.8.0",
-    "hardhat": "^2.18.2",
+    "ethers": "^6.7.1",
+    "hardhat": "^2.17.4",
```

Comment:
^ Do we need to downgrade ?

Reply:
This was left from zk deployment. Will update.

Reply:
@0xKurt just calling this out